Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -64,7 +64,23 @@ public class NetworkConnection : IDisposable
         /// <summary>
         /// The NetworkIdentity for this connection.
         /// </summary>
-        public NetworkIdentity playerController { get; internal set; }
+        [Obsolete("renamed to identity")]
```

Comment:
```suggestion
        [Obsolete("Use NetworkConnection.identity instead")]
```
make it more obvious